packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -608,7 +607,7 @@ - (NSDictionary *)encodingMap
       NSURLSessionDownloadTask *downloadTask = [downloadTasks firstObject];
       if (downloadTask) {
         [downloadTask cancelByProducingResumeData:^(NSData * _Nullable resumeData) {
-          NSString *data = [[NSString alloc] initWithData:resumeData encoding:NSUTF8StringEncoding];
```

Comment:
Could you please add an explanatory comment here and in the previous hunk that this option is the only one with which this code works?